Assignments/stu1/data-access.js

```diff
@@ -7,7 +7,7 @@ const { Pool } = require("pg");
 const GET_PERSON = "select * from person where person_id = $1"
 const GET_BOOK = "select bs.book_store_id, bs.book_store_name, b.title from book_store bs join book_store_book bb on bb.book_store_id = bs.book_store_id join book b on  b.book_id = bb.book_id where bs.book_store_id = $1"
 const UPDATE_PERSON = "update person set first_name = $2 where person_id = $1 returning person_id;"
-
+const ADD_BOOK = "insert into book_store (book_store_name) values ($1)returning book_store;"
```

Comment:
I think the end of this statement should be: `returning book_store_id;`, right?  Because we want to return the new `book_store` id.